### PR TITLE
adding files

### LINKED ISSUE #72: Add pictured examples to Modified LAR Instructions.

### PR DESCRIPTION
**Pull request by MichaelFurey17**
Close #72

### CODE REVIEW

File: documentation_resources/UsingModifiedLar.md [hunk 1]
```diff
@@ -3,6 +3,25 @@ Modified LAR is the public-facing set of the Loan Application Register (LAR) dat
 
 *Note concerning schemas*: Blanks are when a data field contains no entry. Blanks are valid enumerations for some data fields, for more information see the [Filing Instruction Guide](https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2018-hmda-fig-2018-hmda-rule.pdf)
 
+### Opening Modified LAR Text Files With Excel
+Modified LAR is available as a pipe-delimited text file with the .txt extension and has no header row. To read this file in Excel, the following steps would need to be taken.
+
+1. Navigate to the Modified Loan/Application Register (LAR) page [here](https://ffiec.cfpb.gov/data-publication/modified-lar/2018). Enter the institution name in the search box, and click `Download Modified LAR`. 
+2. Open Excel and select the modified LAR file in the Downloads folder.
```
Comment (Kibrael) [inline documentation_resources/UsingModifiedLar.md]: Is there a step where the user needs to open the modified LAR file in Excel?

File: documentation_resources/UsingModifiedLar.md [hunk 2]
```diff
@@ -3,6 +3,25 @@ Modified LAR is the public-facing set of the Loan Application Register (LAR) dat
...
+2. Open Excel and select the modified LAR file in the Downloads folder. 
+![alt text](https://github.com/cfpb/HMDA_Data_Science_Kit/tree/master/documentation_resources/example_images/mlar_tutorial_images/Step2.JPG)
+3. A window will open that tells Excel how to read the file. Select `Delimited` in the first window and click `Next.` 
+![alt text](https://github.com/cfpb/HMDA_Data_Science_Kit/tree/master/documentation_resources/example_images/mlar_tutorial_images/Step3.JPG)
+4. In the next window, specify the pipe delimiter. Select `Other` under the `Delimiters` column and place a pipe (`|`) character in the text box field. Click `Next.`
+![alt text](https://github.com/cfpb/HMDA_Data_Science_Kit/tree/master/documentation_resources/example_images/mlar_tutorial_images/Step4.JPG)
+5. In the final window, select `General` for data type, and select `Finish.` 
+At this point, the data will populate into their own cells.
+![alt text](https://github.com/cfpb/HMDA_Data_Science_Kit/tree/master/documentation_resources/example_images/mlar_tutorial_images/Step5.JPG)
+6. To begin adding a header, highlight the top row of data, right click, and select `Insert` to add a blank row.
```
Comment (Kibrael) [inline documentation_resources/UsingModifiedLar.md]: "to add a header"

File: documentation_resources/UsingModifiedLar.md (hunk 1 above)
Comment (MichaelFurey17) [reply documentation_resources/UsingModifiedLar.md]: I decided to have the user begin with Excel first. Originally, I gave instructions to right click the download and "open with" Excel. The "open with" function for downloads can vary between browsers and operating systems, and so I decided that starting with Excel provides a way to explain this process regardless of operating system or browser.

Comment (Kibrael) [reply documentation_resources/UsingModifiedLar.md]: Right, I get that.

"Step 2. Open Excel and select the modified LAR" needs a clarification.

Users would need to click the "File" menu and select "Open" prior to choosing the MLAR in the downloads folder.

Comment (MichaelFurey17) [reply documentation_resources/UsingModifiedLar.md]: Right - I can add that part. Thanks!

**Pull request merged**